src/main/resources/public/js/wallets.js

```diff
@@ -324,16 +324,14 @@ function payInvoice(invoice, contract, payButton) {
                         function (index, row) {
                             if ($(row).find("td:eq(0)").text() == json.paid) {
                                 var timestamp = json.payment.timestamp;
-                                var failMessage;
+                                var failMessage = "Something went wrong, please try again.";
                                 if(timestamp.length > 0) {
                                     timestamp = timestamp.split('T')[0];
-                                    failMessage = timestamp + ": " + json.payment.failReason;
-                                } else {
-                                    failMessage = json.payment.failReason;
+                                    failMessage = timestamp + ": " + failMessage;
                                 }
                                 var status = "Payment failed " + "<i class='fa fa-exclamation-triangle fa-lg' style='color:red;' aria-hidden='true' "
                                     + "data-toggle='tooltip' data-placement='top' "
-                                    + "data-original-title=\"" + failMessage.replaceAll('"', '\'') + "\">"
+                                    + "data-original-title=\"" + failMessage + "\">"
```

Comment:
@criske The failMessage should be "Something went wrong, please try again" only for payments with status ERROR. If the status is FAILED, then we should display the actual failReason :D 

If I understand correctly, your change will make the failMessage generic for both ERROR and FAILED statuses.